serverless.ts

```diff
@@ -0,0 +1,57 @@
+import { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'product-service',
+    // app and org for use with dashboard.serverless.com
+    // app: your-app-name,
+    // org: your-org-name,
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true
+    }
+  },
+  // Add the serverless-webpack plugin
+  plugins: ['serverless-webpack'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    region: 'eu-west-1',
+    profile: 'mentoring',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+  },
+  functions: {
+    getProductsList: {
+      handler: 'services/getProductsList.handler',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'products',
+          }
+        }
+      ]
+    },
+    getProductById: {
+      handler: 'services/getProductById.handler',
```

Comment:
Looks like you are referencing to multiple services

But the thing is -- 1 servereless = 1 Service

1 Service might contain 2 Lambdas

1 Lambda != Service